projects/fsastorefrontlib/src/core/change-request/facade/change-request.service.ts

```diff
@@ -66,19 +64,17 @@ export class ChangeRequestService {
     );
   }
 
-  loadChangeRequest() {
+  loadChangeRequest(requestId: string) {
     this.userIdService
       .getUserId()
       .pipe(take(1))
       .subscribe(occUserId => {
-        if (this.requestId) {
-          this.store.dispatch(
-            new fromAction.LoadChangeRequest({
-              userId: occUserId,
-              requestId: this.requestId,
-            })
-          );
-        }
+        this.store.dispatch(
+          new fromAction.LoadChangeRequest({
+            userId: occUserId,
+            requestId,
```

Comment:
We should either use shorthand or full a object. We shouldn't combine it like this, it can be confusing to read.